synapse/event_auth.py

```diff
@@ -15,7 +15,18 @@
 
```

Comment:
Some discussion in `#synapse-dev` about this PR: https://matrix.to/#/!vcyiEtMVHIhWXcJAfl:sw1v.org/$oaV7wGmCQOD-pByNgH6x1YhrVrqHBnEuKD76QQ7bClU?via=matrix.org&via=element.io&via=beeper.com

Reply:
Sorry, wrong PR